src/services/housing.js

```diff
@@ -35,7 +35,7 @@ import './user'; // Needed for typedef of StudentProfileInfo
  * @property {DateTime} [BirthDate] The birthday of this applicant (only visible to housing admin)
  * @property {Number} [Age] The age of the student (in years) (only visible to housing admin)
  * @property {String} [Class] Class
- * @property {String} OffCampusProgram The name of department of this applicant's off-campus program, or 'None'
+ * @property {String} OffCampusProgram The name of department of this applicant's off-campus program, or '' (empty string)
```

Comment:
This confused me just now, I thought it was a typo, so maybe this will make it easier to read?
```suggestion
 * @property {String} OffCampusProgram The name of department of this applicant's off-campus program, or ( '' ) (i.e., an empty string)
```

Reply:
I personally find the parentheses around `''` more confusing. Maybe this would be the clearest?:
```suggestion
 * @property {String} OffCampusProgram The name of department of this applicant's off-campus program, or  an empty string
```

Reply:
Feel free to commit your suggestion here